awscli/customizations/emr/helptext.py

```diff
@@ -56,6 +56,18 @@
     ' config file using the "aws configure set" command: --service-role, --log-uri,'
     ' and InstanceProfile and KeyName arguments under --ec2-attributes.')
 
+DESCRIBE_CLUSTER_DESCRIPTION = (
+    'Provides  cluster-level details including status, hardware'
+    ' and software configuration, VPC settings, bootstrap'
+    ' actions, instance groups and so on.'
+    '\nPermissions needed for describe-cluster include:\n'
+    '\n"elasticmapreduce:ListBootstrapActions",\n'
```

Comment:
Instead of using newlines, could we instead use `<ul>`/`<li>` elements to build up a bulleted list? We should also probably wrap these permissions with `<code>` elements (e.g.  `<code>elasticmapreduce:ListBootstrapActions</code>`) instead of quotes. Both of these changes will make it more consistent with other areas where we need to list elements. For example, we should probably copy the same approach from this doc string: https://github.com/aws/aws-cli/blob/develop/awscli/customizations/s3/subcommands.py#L157